reconstruction/cnd/src/main/java/org/jlab/rec/cnd/constants/Parameters.java

```diff
@@ -52,6 +52,8 @@ public class Parameters {
 
 	public static double EThresh = 0.1 ;                                                       // Min reconstructed deposited energy threshold for a good event
 
+	public static double DeltaZDH = 2;                                                         // in cm, maximum absolute value of difference between hit z and double hit Z
```

Comment:
as discussed, these parameters should be moved to a CCDB table. See https://clasweb.jlab.org/cgi-bin/ccdb/show_request?request=/calibration/cnd/double_hits:0:default:2023-02-26_21-54-55

Reply:
changes are made, th ccdb table is now read insted of hardcoded parameters